### DISCUSSION TITLE
Non US keyboard help request

### DISCUSSION THREAD

**Discussion by shichou** [atom/atom#22951] [Q&A]
Hi,

Could anyone point me to where I can find information about how to change the default keyboard settings to a UK keyboard?  Googling led me to a keyboard-localization package, but when I tried to install it, the package description is labelled 'deprecated' and says:  

_:boom: The atom team has managed to integrate support for international keyboards into atom v1.12.x core. :tada:
If you're on atom >= 1.12.x this package shouldn't be neccessary any more._

However, there weren't obvious instructions as to what to do instead.  (There are links to in-depth discussions about the changes, and some of the difficulties with doing so, and troubleshooting, etc., but I can't find a simple guide to how to actually set the keyboard to a UK layout without individually changing the keymappings.  It's possible I don't know the right keywords to search.)  

I'd be very grateful if someone could either tell me how to do this or, even better, give me a heads-up as to where to find this in the documentation so I can start finding my way around a bit better.  (I'm still very new to Atom and am coming from more basic editors like pluma and gedit.  Loving it so far, except for the US keyboard default.)

Thanks so much.

**Comment by JavaShipped** [comment]
I want to bump this as I'm having issues as well. No other application on my PC uses the US keypress layout, but atom does. I need to manually change it to the UK layout.
